### DISCUSSION TITLE
Diferencias con archivos de output entre partes

### DISCUSSION THREAD

**Discussion by GitCross135** [IIC2133-PUC/2024-1#187] [Tarea 3] [answered]
Hola, normalmente en los tests, los archivos de output dejan una línea en blanco al final (en las partes de nyctalus y find_groups) pero en trash no. ¿Qué hago? ¿Dejo todos los archivos sin la línea final vacía o la dejo en todos menos en Trash? o cómo.

Así se imprime normalmente:
![image](https://github.com/IIC2133-PUC/2024-1/assets/130419134/bdd06035-c2c1-4731-a3b4-c29bec373855)

Así son los archivos de los tests de Trash:
![image](https://github.com/IIC2133-PUC/2024-1/assets/130419134/87071fbb-8fb5-4c1e-ab1f-0c13e88fcf53)

**Comment by benjavicente** [comment, marked as answer]
Deja el salto de línea al final (que quede una linea vacía). De todas formas, no debería afectar con `diff -b`, que ignora pequeños cambios de espacios.
